hello-world-module-methods/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+greet.sayHey('Jinho');
+greet. sayBye();
```

Comment:
This file is also not necessary since you have another index.js already calling the methods in your greet.js.